test/oscd-action-pane.spec.ts

```diff
@@ -0,0 +1,149 @@
+import { expect, fixture, html } from '@open-wc/testing';
+
+import '../src/oscd-action-pane.js';
+import { OscdActionPane } from '../src/oscd-action-pane.js';
+
+describe('oscd-action-pane', () => {
+  let element: OscdActionPane;
+  const label = 'test label';
+
+  beforeEach(async () => {
+    element = await fixture(
+      html`<oscd-action-pane
+        header="test header"
+        label="${label}"
+      ></oscd-action-pane>`
+    );
+    await element.updateComplete;
+  });
+
+  it('looks like the latest snapshot', () => {
+    expect(element).shadowDom.to.equalSnapshot();
+  });
+
+  it('renders the header as <h1> per default', () => {
+    expect(element.shadowRoot?.querySelector('h1')).to.exist;
```

Comment:
can you change this test to the following:
```
const header: HTMLElement = element.shadowRoot.querySelector(`[title="${label}"]`);
expect(header).to.exist;
expect(header.tagName).toEqual('h1');
```

With this, you're not asserting on something that don't exist